src/file.rs

```diff
@@ -1,13 +1,46 @@
+use std::str::FromStr;
+
 use crate::{
     common::timestamp,
     traits::{HasPath, TimestampId, Validatable},
     APP_PATH,
 };
+use mime::Mime;
 use serde::{Deserialize, Serialize};
 
+use url::Url;
 #[cfg(feature = "openapi")]
 use utoipa::ToSchema;
 
+const MIN_NAME_LENGTH: usize = 1;
+const MAX_NAME_LENGTH: usize = 255;
+const MAX_SRC_LENGTH: usize = 1024;
+const MAX_SIZE: i64 = 10_000_000; // 10 MB
```

Comment:
Are we using decimal MB?

To avoid confusion maybe best to do `2^20 B` 
```
const MAX_SIZE: i64 = 10 * (1 << 20);
```
or `10_485_760`